client/me/account-password/index.jsx

```diff
@@ -90,7 +92,7 @@ module.exports = React.createClass( {
 					debug( 'Error saving password: ' + JSON.stringify( error ) );
 
 					// handle error case here
-					notices.error( this.translate( 'There was a problem saving your password. Please, try again.' ) );
+					this.props.errorNotice( this.translate( 'There was a problem saving your password. Please, try again.' ) );
```

Comment:
fwiw, I also like @dmsnell's suggestion. I think destructuring is the bee's knees.